### ISSUE TITLE
TooltipDefinition.js accessibility issue

### ISSUE THREAD

**Issue by harshitshah436** [carbon-design-system/carbon#7688]
## Environment

> Operating system: Windows

> Browser: Chrome

> Automated testing tool and ruleset: VS Code & Selenium automated testing

## Detailed description

> What version of the Carbon Design System are you using?: 10.10.3

> What did you expect to happen?: No accessibility error

> What happened instead?: Getting an error with the title "Form control element <button> has no associated label"

<img width="876" alt="tooltip-definition_error" src="https://user-images.githubusercontent.com/14295316/106200495-11ac9600-6185-11eb-8c5e-2d355217f94e.png">

## Steps to reproduce the issue

1. Use the tooltipDefinition component

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

Here's the CodeSandbox link: https://codesandbox.io/s/peaceful-bassi-615ns?file=/src/index.js

**Comment by tw15egan** [comment]
Thanks for creating an issue. However, I'm still unable to reproduce this a11y violation. Here's what I'm seeing (just violations relating to the CodeSandbox instance itself): <img width="796" alt="Screen Shot 2021-01-28 at 1 52 05 PM" src="https://user-images.githubusercontent.com/11928039/106203306-0864fe80-6170-11eb-88e5-47a4110b0cc5.png">

**Comment by harshitshah436** [comment]
yeah, even I don't seem to find that issue. Our automated tests run on electron VS Code's inbuilt browser. So that may be the issue. For now, what do you suggest? just ingore and close this defect?

**Comment by tw15egan** [comment]
Yeah since we're unable to reproduce there isn't any action we can take. I wonder if it would be worth raising an issue on whatever testing suite library you are using that is flagging the error?

**Comment by dakahn** [comment]
Closing this as resolved. Thanks for submitting though 🏄🏽
